README.md

```diff
@@ -7,7 +7,8 @@ Built With:
 Html
 css
 
-Live Demo (if available)
+Live Demo:
+https://leonard33.github.io/mobile-portfolio-skeleton/
 
 Authors :
 Leonard Kombo
```

Comment:
- [ ] Kindly as the previous reviewer asked, you have to use this [template](https://github.com/microverseinc/readme-template) provided by Microverse, take it cope, and replace your readme with it, and customize it to match your project. e.g. add your link to the `Live Demo` section.

Reply:
Hi. the link is provided there in the README file you can click and be redirected to the project demo. Is there something I don't understand you elaborate further?

Reply:
Hey I am waiting for your answer?

Reply:
@leonard33 Hey, sorry for being late, but mention me to get a notification.
the link is for a template take it cope and replace it with your readme.